app/src/main/java/com/eveningoutpost/dexdrip/Home.java

```diff
@@ -3016,6 +3018,7 @@ public boolean onOptionsItemSelected(MenuItem item) {
                 break;
             case R.id.action_open_watch_settings:
                 startService(new Intent(this, WatchUpdaterService.class).setAction(WatchUpdaterService.ACTION_OPEN_SETTINGS));
```

Comment:
It never had mattered as there where only the two options in the menu and this was the second one.
Good example why to write it anyway - to not fall into the trap later :) 
Nice catch